src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx

```diff
@@ -127,6 +127,10 @@ export const MyTimeline: React.FC<MyTimelineProps> = ({ day, myTasks, taskDetail
         //   setOpenReschedule({ ...task });
         // }
       } else if (actionName === "Done") {
+        const markDone = !!taskDetails[task.goalid]?.completedTodayIds.includes(task.taskid);
+
+        if (markDone) return;
+
```

Comment:
you can remove this line break.

Reply:
Hi @Tushar-4781 are you saying to remove the line break that is between these two lines of code or to remove these both lines?